jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -1225,7 +1225,7 @@ private static boolean isIncluded(HttpServletRequest request)
 
     private static boolean isDefaultMapping(HttpServletRequest req)
     {
-        return req.getHttpServletMapping().getMappingMatch() == MappingMatch.DEFAULT;
+        return isIncluded(req) || req.getHttpServletMapping().getMappingMatch() == MappingMatch.DEFAULT;
```

Comment:
I don't think this is sufficient/correct.     A `DefaultServlet` mapped to some other location and/or name can be the target of an include.

I'm beginning to wonder if we should rename this to be `ResourceServlet` with this method returning false, and then just have an extended version of it called `DefaultServlet` that returns true for this method?

Reply:
hmmm the problem with the idea above is that the same servlet instance might be both at "/" and "/somepath/*".
Oh, what happens if it is mapped to "*.suffix"? Do we use the right part of the path then?

Reply:
Maybe we just use the `pathInfo` if the `servletPath` is empty?